pkg/logql/expr.y

```diff
@@ -233,8 +239,12 @@ labelFilter:
     | labelFilter OR labelFilter { $$ = labelfilter.NewOr($1, $3 ) }
     ;
 
+unitFilter:
```

Comment:
I'm curious why do you need to use an intermediary type ?

Reply:
I thought it made the label filter easier to read and extend. Eg if we support over units.

Reply:
sounds good !